diff --git a/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt b/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
index b01cf74..0dbf393 100644
--- a/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
+++ b/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
@@ -1,6 +1,3 @@
-# Require CMake 3.10. If available, use the policies up to CMake 3.22.
-cmake_minimum_required (VERSION 3.10...3.22)
-
 project (benchmark VERSION 1.8.0 LANGUAGES CXX)
 
 option(BENCHMARK_ENABLE_TESTING "Enable testing of the benchmark library." ON)
