diff --git a/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt b/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
index 0dbf393..0ceff17 100644
--- a/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
+++ b/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
@@ -53,6 +53,10 @@ if(MSVC)
     endif()
 endif()
 
+# BLT EDIT START
+set(CMAKE_BUILD_TYPE ${_BLT_GBENCHMARK_COMPILE_FLAGS})
+# BLT EDIT END
+
 set(ENABLE_ASSEMBLY_TESTS_DEFAULT OFF)
 function(should_enable_assembly_tests)
   if(CMAKE_BUILD_TYPE)
