diff --git a/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt b/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
index 0dbf393..5870dfc 100644
--- a/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
+++ b/thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt
@@ -192,6 +192,9 @@ else()
     # See #631 for rationale.
     add_cxx_compiler_flag(-wd1786)
   endif()
+  if (CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
+    add_cxx_compiler_flag(-Wno-tautological-compare)
+  endif()
   # Disable deprecation warnings for release builds (when -Werror is enabled).
   if(BENCHMARK_ENABLE_WERROR)
       add_cxx_compiler_flag(-Wno-deprecated)
