diff --git a/thirdparty_builtin/googletest-1.16.0/CMakeLists.txt b/thirdparty_builtin/googletest-1.16.0/CMakeLists.txt
index ca4aceb..93b665a 100644
--- a/thirdparty_builtin/googletest-1.16.0/CMakeLists.txt
+++ b/thirdparty_builtin/googletest-1.16.0/CMakeLists.txt
@@ -11,7 +11,6 @@ endif()
 enable_testing()
 
 include(CMakeDependentOption)
-include(GNUInstallDirs)
 
 # Note that googlemock target already builds googletest.
 option(BUILD_GMOCK "Builds the googlemock subproject" ON)
