diff --git a/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake b/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake
index 580ac1c..c01906a 100644
--- a/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake
+++ b/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake
@@ -136,6 +136,8 @@ macro(config_compiler_and_linker)
     set(cxx_no_exception_flags "+noeh -DGTEST_HAS_EXCEPTIONS=0")
     # RTTI can not be disabled in HP aCC compiler.
     set(cxx_no_rtti_flags "")
+  elseif (CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
+    set(cxx_base_flags "-Wno-tautological-compare")
   endif()
 
   # The pthreads library is available and allowed?
