From 170e1667b91adff6f58aac0bf04aa2d68f57b68d Mon Sep 17 00:00:00 2001
From: David Beckingsale <davidbeckingsale@gmail.com>
Date: Thu, 1 Feb 2018 11:09:04 -0800
Subject: [PATCH] Allow user-override for death tests

---
 .../googletest/include/gtest/internal/gtest-port.h                      | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/internal/gtest-port.h b/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/internal/gtest-port.h
index 01ad5da..1c4e1ba 100755
--- a/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/internal/gtest-port.h
+++ b/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/internal/gtest-port.h
@@ -802,6 +802,7 @@ using ::std::tuple_size;
 # endif  // !GTEST_OS_WINDOWS_MOBILE && !GTEST_OS_SYMBIAN
 #endif  // GTEST_HAS_STREAM_REDIRECTION
 
+#ifndef GTEST_HAS_DEATH_TEST
 // Determines whether to support death tests.
 // Google Test does not support death tests for VC 7.1 and earlier as
 // abort() in a VC 7.1 application compiled as GUI in debug config
@@ -813,6 +814,7 @@ using ::std::tuple_size;
      GTEST_OS_OPENBSD || GTEST_OS_QNX || GTEST_OS_FREEBSD || GTEST_OS_NETBSD)
 # define GTEST_HAS_DEATH_TEST 1
 #endif
+#endif
 
 // Determines whether to support type-driven tests.
 
-- 
2.10.1 (Apple Git-78)

