diff --git a/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h b/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h
index b4fa3f0..a4a9f10 100644
--- a/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h
+++ b/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h
@@ -612,6 +612,7 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 #endif  // !GTEST_OS_WINDOWS_MOBILE
 #endif  // GTEST_HAS_STREAM_REDIRECTION
 
+#ifndef GTEST_HAS_DEATH_TEST
 // Determines whether to support death tests.
 // pops up a dialog window that cannot be suppressed programmatically.
 #if (GTEST_OS_LINUX || GTEST_OS_CYGWIN || GTEST_OS_SOLARIS ||             \
@@ -626,6 +627,7 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 #define GTEST_HAS_DEATH_TEST 1
 #endif  // GTEST_HAS_FILE_SYSTEM
 #endif
+#endif
 
 // Determines whether to support type-driven tests.
 
