diff --git a/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/gtest-printers.h b/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/gtest-printers.h
index 4deaad0..3073ff1 100755
--- a/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/gtest-printers.h
+++ b/thirdparty_builtin/googletest-master-2018-02-01/googletest/include/gtest/gtest-printers.h
@@ -634,6 +634,10 @@ inline void PrintTo(absl::string_view sp, ::std::ostream* os) {
 }
 #endif  // GTEST_HAS_ABSL
 
+#if GTEST_LANG_CXX11
+inline void PrintTo(std::nullptr_t, ::std::ostream* os) { *os << "(nullptr)"; }
+#endif  // GTEST_LANG_CXX11
+
 #if GTEST_HAS_TR1_TUPLE || GTEST_HAS_STD_TUPLE_
 // Helper function for printing a tuple.  T must be instantiated with
 // a tuple type.
