diff --git a/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h b/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h
index b00aa77..9182767 100644
--- a/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h
+++ b/thirdparty_builtin/googletest/googletest/include/gtest/internal/gtest-port.h
@@ -859,7 +859,10 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 #define GTEST_NO_INLINE_
 #endif
 
-#if GTEST_HAVE_ATTRIBUTE_(disable_tail_calls)
+// BLT EDIT: xlc compilers fail trying to set the flag
+#if defined(__ibmxl__)
+#define GTEST_NO_TAIL_CALL_
+#elif GTEST_HAVE_ATTRIBUTE_(disable_tail_calls)
 // Ask the compiler not to perform tail call optimization inside
 // the marked function.
 #define GTEST_NO_TAIL_CALL_ __attribute__((disable_tail_calls))
