diff --git a/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake b/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake
index c683178..a1865a0 100644
--- a/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake
+++ b/thirdparty_builtin/googletest-1.16.0/googletest/cmake/internal_utils.cmake
@@ -94,7 +94,7 @@ macro(config_compiler_and_linker)
     set(cxx_base_flags "-Wall -Wshadow -Wconversion -Wundef")
     set(cxx_exception_flags "-fexceptions")
     set(cxx_no_exception_flags "-fno-exceptions")
-    set(cxx_strict_flags "-W -Wpointer-arith -Wreturn-type -Wcast-qual -Wwrite-strings -Wswitch -Wunused-parameter -Wcast-align -Winline -Wredundant-decls")
+    set(cxx_strict_flags "-W -Wpointer-arith -Wreturn-type -Wcast-qual -Wwrite-strings -Wswitch -Wunused-parameter -Wcast-align -Wredundant-decls")
     set(cxx_no_rtti_flags "-fno-rtti")
     if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
       set(cxx_strict_flags "${cxx_strict_flags} -Wchar-subscripts")
